Components/FirstRunWizSteps/Step4.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { StyleSheet, Text, View } from "react-native";
+import { useFormikContext } from "formik";
+import { FirstRunValues } from "../../customTypes";
+import Errors from "../Errors";
+import { checkboxHitSlop } from "../../Helpers/helpers";
+import Checkbox from "expo-checkbox";
+
+export default function Step4() {
+    const { values, errors, setFieldValue } =
+        useFormikContext<FirstRunValues>();
+    console.log(errors);
+    return (
+        <View>
+            <Text style={styles.header}>Available Days</Text>
+
+            <Text style={styles.sectionInfo}>
+                Check which days of the week are you available.
```

Comment:
```suggestion
                Check the days of the week that you are available.
```
Maybe something like that?